easybuild/easyconfigs/g/GDAL/GDAL-3.2.1-foss-2020b.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '3.2.1'
+
+homepage = 'https://www.gdal.org'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['GDAL-3.0.0_fix-python-CC-CXX.patch']
+checksums = [
+    '6c588b58fcb63ff3f288eb9f02d76791c0955ba9210d98c3abd879c770ae28ea',  # gdal-3.2.1.tar.xz
+    '223a0ed1afb245527d546bb19e4f80c00f768516ab106d82e53cf36b5a1a2381',  # GDAL-3.0.0_fix-python-CC-CXX.patch
+]
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('netCDF', '4.7.4'),
+    ('expat', '2.2.9'),
+    ('GEOS', '3.9.1'),
+    ('SQLite', '3.33.0'),
+    ('libxml2', '2.9.10'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.5'),
+    ('JasPer', '2.0.14'),
+    ('LibTIFF', '4.1.0'),
+    ('zlib', '1.2.11'),
+    ('cURL', '7.72.0'),
+    ('PCRE2', '10.35'),
+    ('PROJ', '7.2.1'),
+    ('libgeotiff', '1.6.0'),
+    ('SciPy-bundle', '2020.11'),
+    ('HDF5', '1.10.7'),
+]
+
+preconfigopts = "sed -e 's/-llapack/\$LIBLAPACK/g' -i.eb configure && "
+configopts = '--with-expat=$EBROOTEXPAT --with-libz=$EBROOTLIBZ'
+configopts += ' --without-hdf4 --with-hdf5=$EBROOTHDF5 --with-netcdf=$EBROOTNETCDF'
```

Comment:
@boegel we recently added `hdf4` support. Any reason not to keep that?

Reply:
Ah, I overlooked that... Will fix.